core/src/main/java/cucumber/runtime/arquillian/ArquillianCucumber.java

```diff
@@ -69,13 +76,13 @@
 public class ArquillianCucumber extends Arquillian {
     private static final Logger LOGGER = Logger.getLogger(ArquillianCucumber.class.getName());
 
-    private static final String RUN_CUCUMBER_MTD = "____Cucumber_Runner_Not_A_Test";
```

Comment:
open to name suggestion but runCucumber is a name a user can likely use in cukespace context and it is not forbidden to have @Test public void runCucumber(); test method.

Best would be for sure to just remove it and call it but needs more code in the runner